projectDocs/dev/codingStandards.md

```diff
@@ -106,3 +106,10 @@ In summary:
 Any module level variables should be prefixed with and underscore and be encapsulated, e.g. via getters and setters.
 - Avoid code which executes at import time.
 Instead use initializer functions.
+
+### Docstrings
+

```

Comment:
1. In the link, the types are specified. IMO, we should indicate here explicitly that, in NVDA's codebase, we use type annotation instead of specifying the type in the docstring.
2. In addition, although all the information is already in the link, could we have here an example?

Reply:
I think for Sphinx including types is important. If we are going to auto-generate API documentation it should include type information. There are tools to auto-generate docstrings with types using Sphinx from type annotations

Reply:
Thoughts @leonardder?

Reply:
I think we should indeed avoid types in doc strings and use type annotations instead. There is [this](https://github.com/agronholm/sphinx-autodoc-typehints) module that covers that. I have no experience with it whatsoever.
So long story short, agreed with @CyrilleB79.

Reply:
Added a note re: type hints